src/components/Dashboard/Dashboard.module.scss

```diff
@@ -1,42 +1,56 @@
-$base-unit: 4px;
-$base-font-size: 0.3rem;
 $color-blue: #041187;
 $color-red: #e30062;
 $color-green: #85da6b;
-$color-disabled: #e5e7eb;
+$color-disabled: #7c7e82;
 $color-white: #fff;
 $color-black: #000;
-$diff-margin: $base-unit * 15;
-$similar-margin: $base-unit * 8;
-$section-padding-top-bottom: $base-unit * 30;
-$section-padding-left-right: $base-unit * 25;
 
 .container {
-    margin: $diff-margin auto;
-    width: 85%;
+    margin-top: 40px;
+    display: flex;
+    justify-content: center;
+    line-height: 2rem;
+    align-items: center;
 }
 
-.searchLabel {
-    font-size: $base-font-size * 5;
-    padding-right: $base-unit * 8;
-    color: $color-blue;
+.searchBar {
```

Comment:
Also, please give some padding around the search bar and the button.

Reply:
Please check out the video there is space between the button and the search bar, an increased amount of space wasn't looking good.

Reply:
![image](https://github.com/Real-Dev-Squad/website-status/assets/51514137/a6481e11-6b39-482e-a042-71f7cbfbff5d)
I didn't mean between the input and button, I meant around it. Please check the SS and find the space marked in blue

Reply:
Oh, thankyou for pointing it out, I have fixed it now and updated the video please verify it